doc/version_history.rst

```diff
@@ -6,6 +6,20 @@
 Version History
 ###############
 
+v0.31.0
+-------
+
+* `CommandTelemetryClient`: inherit from `lsst.ts.tcpip.Client`, which requires ts_tcpip 1.0.
+* Stop exporting symbols from ts_tcpip, except ``LOCAL_HOST``, which is still used by ts_mtrotator.
```

Comment:
I think `LOCAL_HOST` is used by `ts_m2com` and `ts_mtdome` as well.

Reply:
Many packages use LOCAL_HOST from ts_tcpip, but I would be surprised if any packages other than ts_mthexrotcomm or ts_mtrotator use it from ts_hexrotcomm.